src/licenses/mod.rs

```diff
@@ -216,6 +216,17 @@ impl Gatherer {
     }
 }
 
+#[cfg(unix)]
+fn is_fifo(ft: std::fs::FileType) -> bool {
+    use std::os::unix::fs::FileTypeExt;
+    ft.is_fifo()
+}
+
+#[cfg(not(unix))]
```

Comment:
Why does this exist? `is_fifo` only seems to be used in an a `cfg(unix)` block.

Reply:
We might also just want to remove the `cfg(unix)` block below.

Reply:
Because is_fifo is only available on unix, not windows.

Reply:
Oh I see you actually made an if block below and not cfg'ed out the whole block.

Reply:
Actually that would be better